shell/src/main/java/alluxio/cli/fs/command/CheckConsistencyCommand.java

```diff
@@ -109,7 +110,8 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
   private void runConsistencyCheck(AlluxioURI path, boolean repairConsistency) throws
       AlluxioException, IOException {
     List<AlluxioURI> inconsistentUris =
-        checkConsistency(path, CheckConsistencyPOptions.getDefaultInstance());
```

Comment:
FileSystem should do merging.

Reply:
The checkConsistency function uses the underlying client, not the public client API. So it still needs to merge options.